src/main/java/org/jabref/model/database/BibDatabase.java

```diff
@@ -116,7 +116,7 @@ public boolean containsEntryWithId(String id) {
     }
 
     public ObservableList<BibEntry> getEntries() {
-        return FXCollections.unmodifiableObservableList(entries);
+        return entries;
```

Comment:
Is there a reason why the entries are all wrapped in an Unmodiefable collection?

Reply:
Yes, so that nobody can add entries to the database without calling `insertEntry` which also does some bookkeeping (id list, tex free version, ...). I think the general principle is that objects should never expose their inner data.